### PR TITLE
Make traces more descriptive in a host and ipc

### PR DESCRIPTION
**Pull request by ktrzcinx**

### CODE REVIEW

File: src/ipc/ipc.c [hunk 1]
```diff
@@ -401,15 +401,15 @@ int ipc_comp_connect(struct ipc *ipc,
 	/* check whether the components already exist */
 	icd_source = ipc_get_comp_by_id(ipc, connect->source_id);
 	if (!icd_source) {
-		tr_err(&ipc_tr, "ipc_comp_connect(): components already exist, connect->source_id = %u",
-		       connect->source_id);
+		tr_err(&ipc_tr, "ipc_comp_connect(): source components already not exist, source_id = %u sink_id = %u",
```
Comment (lgirdwood) [inline src/ipc/ipc.c:406]: "source component does not exist" and yes I am glad the strings are in English and not in Polish, Romanian or Mandarin !

File: src/ipc/ipc.c [hunk 2]
```diff
@@ -401,15 +401,15 @@ int ipc_comp_connect(struct ipc *ipc,
...
+		       connect->source_id, connect->sink_id);
 		return -EINVAL;
 	}
 
 	icd_sink = ipc_get_comp_by_id(ipc, connect->sink_id);
 	if (!icd_sink) {
-		tr_err(&ipc_tr, "ipc_comp_connect(): components already exist, connect->sink_id = %u",
-		       connect->sink_id);
+		tr_err(&ipc_tr, "ipc_comp_connect(): sink components already not exist, source_id = %d sink_id = %u",
```
Comment (lgirdwood) [inline src/ipc/ipc.c:413]: "sink component does not exist"

**Review by lgirdwood — CHANGES_REQUESTED** [review]
Just minor string changes otherwise is good.

File: src/ipc/ipc.c (hunk 1 above)
Comment (ktrzcinx) [reply src/ipc/ipc.c]: I've always heard learning new languages is valuable and allows to reach many things 😄

**Review by lgirdwood — APPROVED** [review]

File: src/ipc/ipc.c [hunk 3]
```diff
@@ -401,15 +401,15 @@ int ipc_comp_connect(struct ipc *ipc,
 	/* check whether the components already exist */
```
Comment (dbaluta) [inline src/ipc/ipc.c:403]: small typos in commit message: connet, exisct

File: src/ipc/ipc.c [hunk 4]
```diff
@@ -401,15 +401,15 @@ int ipc_comp_connect(struct ipc *ipc,
...
 		return -EINVAL;
 	}
 
 	icd_sink = ipc_get_comp_by_id(ipc, connect->sink_id);
 	if (!icd_sink) {
-		tr_err(&ipc_tr, "ipc_comp_connect(): components already exist, connect->sink_id = %u",
-		       connect->sink_id);
+		tr_err(&ipc_tr, "ipc_comp_connect(): sink components does not exist, source_id = %d sink_id = %u",
+		       connect->sink_id, connect->source_id);
 		return -EINVAL;
```
Comment (dbaluta) [inline src/ipc/ipc.c:415]: should we replace components with component? e.g ``ipc_comp_connect(): sink component does not exist...``?

File: src/audio/host.c [hunk 5]
```diff
@@ -346,6 +346,18 @@ static uint32_t host_get_copy_bytes_normal(struct comp_dev *dev)
 	 */
 	copy_bytes = ALIGN_DOWN(copy_bytes, hd->dma_copy_align);
 
+	if (!copy_bytes) {
```
Comment (dbaluta) [inline src/audio/host.c:349]: @ktrzcinx this is a very useful change!

File: src/ipc/ipc.c (hunk 4 above)
Comment (ktrzcinx) [reply src/ipc/ipc.c:415]: Updated

File: src/ipc/ipc.c (hunk 3 above)
Comment (ktrzcinx) [reply src/ipc/ipc.c:403]: Updated

File: src/audio/host.c (hunk 5 above)
Comment (paulstelian97) [reply src/audio/host.c:349]: Agreed, this should reduce my number of extra debug prints!

**Review by paulstelian97 — APPROVED** [review]

**Review by dbaluta — APPROVED** [review]

**Pull request merged**